package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vscode-rust",
   "displayName": "Rust",
   "description": "Rust language integration for VSCode",
-  "version": "0.4.2",
```

Comment:
Could you explain why you want to increase the version of the extension?

Reply:
Your are right, I shouldn't have done so. I have reverted the change.